src/main/java/frc/robot/subsystems/arm/PrototypeArm.java

```diff
@@ -0,0 +1,145 @@
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.*;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.CANCoder;
+import edu.wpi.first.math.geometry.Translation2d;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+import frc.robot.utils.units.UnitModel;
+
+public class PrototypeArm extends LoggedSubsystem<PrototypeArmLogInputs> {
```

Comment:
I see that both parts of arm have the same kind of computation and functions, maybe consider creating a class for each part of the system? (Like we do in the swerve)